### PR TITLE
Adds Kubernetes Inventory UI

### PR DESCRIPTION
**Pull request by hev**
This PR updates the docs related to Kubernetes UI and makes Kubernetes Inventory more prominent. Includes the following changes.

* Highlights KAI on homepage
* Adds Using Kubernetes Inventory section
* Cleans up the concepts page to focus on installing kai and moves embedded mode to sub-page.

Would still like to add screenshots to the using page but submitting as is for now.

### CODE REVIEW

File: content/docs/overview/concepts/images/inventory/_index.md [hunk 1]
```diff
@@ -1,153 +1,62 @@
...
-Anchore can connect to your runtime environment and discern which images exist in each context (i.e. Kubernetes Namespace).
-
-Currently, supported runtime environments include:
-* Kubernetes
-
-# Kubernetes Runtime Inventory 
 ## Configuration
-Anchore uses a go binary called [kai](https://github.com/anchore/kai) that leverages the [Kubernetes Go SDK](https://github.com/kubernetes/client-go) to reach out and list
-pods in a configurable set of namespaces to determine which images are running.
+Anchore uses a go binary called [kai](https://github.com/anchore/kai) that leverages the [Kubernetes Go SDK](https://github.com/kubernetes/client-go) to reach out and list pods in a configurable set of namespaces to determine which images are running.
```
Comment (Toure) [inline content/docs/overview/concepts/images/inventory/_index.md:8]: nit: Kubernetes Go is a library, should we include it in the docs?

**Review by Toure — CHANGES_REQUESTED** [review]
Just a small nit on one thing. But it looks good.

File: content/docs/overview/concepts/images/inventory/_index.md [hunk 2]
```diff
@@ -1,153 +1,62 @@
...
-- name: docker-desktop
-  user:
-    client-certificate-data: def
-    client-key-data: ghi
-- name: kai-demo
-  user:
-    token: jkl
-```
+## Agent Mode
+The most common way to track inventory is to install [`kai`](https://github.com/anchore/kai). as an agent in your cluster. To do this you will need to configure credentials
```
Comment (zhill) [inline content/docs/overview/concepts/images/inventory/_index.md:13]: Typo with extra period after the link.
```suggestion
The most common way to track inventory is to install [`kai`](https://github.com/anchore/kai) as an agent in your cluster. To do this you will need to configure credentials
```

File: content/_index.html [hunk 3]
```diff
@@ -19,8 +19,8 @@
 
 {{% blocks/section title="Integrations" subtitle="Integrate Anchore with your favorite CI/CD Systems, Orchestration Platforms, and more" color="primary" %}}
 
-{{% blocks/feature svg="icons/kubernetes-logo.svg" title="Kubernetes" %}}
-Install Anchore using the [Helm Chart](https://github.com/anchore/anchore-charts/blob/master/stable/anchore-engine) and make admission decisions based on analysis and policy-based evaluations with our [Kubernetes Admission Controller](https://github.com/anchore/kubernetes-admission-controller)
+{{% blocks/feature svg="icons/kubernetes-logo.svg" href="/docs/using/ui_usage/kubernetes/" title="Kubernetes" %}}
```
Comment (zhill) [inline content/_index.html:22]: This link will 404 in the built site.
```suggestion
{{% blocks/feature svg="icons/kubernetes-logo.svg" href="/current/docs/using/ui_usage/kubernetes" title="Kubernetes" %}}
```

File: content/docs/using/ui_usage/kubernetes/_index.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Kubernetes Inventory"
+linkTitle: "Kubernetes Inventory"
+weight: 1
+---
+Anchore Enterprise allows you to navigate through your Kubernetes clusters to quickly and easy asses your vulnerabilites, apply policies, and take action on them. You'll need to configure your clusters for collection before being able to take advanatage of these features. See our [installation instructions to get setup](/docs/overview/concepts/images/inventory/).
```
Comment (zhill) [inline content/docs/using/ui_usage/kubernetes/_index.md:6]: Ref update here as well

File: content/docs/overview/concepts/images/inventory/_index.md [hunk 5]
```diff
@@ -1,153 +1,62 @@
...
 * For injecting the environment variable, see: `inject_secrets_via_env`
-* For providing your own secret for the Anchore API Password, see: `kai.existing_secret`. `kai` creates it's own secret based on your values.yaml file for key `kai.anchore.password`, but the `kai.existingSecret` key allows you to create your own secret and provide it in the values file.
\ No newline at end of file
+* For providing your own secret for the Anchore API Password, see: `kai.existing_secret`. `kai` creates it's own secret based on your values.yaml file for key `kai.anchore.password`, but the `kai.existingSecret` key allows you to create your own secret and provide it in the values file.
+
+## Agentless mode
+
+In addition to running as agent kai can also be run by Anchore Enterpise in an embedded mode. In this mode `kai` runs within the Anchore Enterprise, and needs a cluster configuration to access the Kubernetes API. This still requires cluster but allows for monitoring without the installation of additional components. See more details about setting up [embedded mode](./embedded/).
```
Comment (zhill) [inline content/docs/overview/concepts/images/inventory/_index.md:49]: Use content ref for link here
```suggestion
In addition to running as agent kai can also be run by Anchore Enterpise in an embedded mode. In this mode `kai` runs within the Anchore Enterprise, and needs a cluster configuration to access the Kubernetes API. This still requires cluster but allows for monitoring without the installation of additional components. See more details about setting up [embedded mode]({{< relref "embedded" >}}). 
```

File: content/docs/overview/concepts/images/inventory/_index.md [hunk 6]
```diff
@@ -1,153 +1,62 @@
...
+
+## CLI and API Inventory Collection
+In situations where you may not be able to have consitent network connection `kai` along with the inventory API can be used to collect and report inventory back to Anchore Enterprise. Inventory collection can also be used to collect other types of containers runtimes however non are officially supported at this time. 
+
+### Usage
+To quickly verify that you are tracking Kubernetes Inventory you can quickly access inventory results from the `anchorectl` with the  command `anchorectl inventory list`. The UI for the Kubernetes allows operators to visually navigate vulnerability resultes, source where vulnerabilties come from and apply their default policy to thier runtime. You'll be able to subscribe to changes for your clusters, quicky asses vulnerabilities and more.
+
+For more details about watching clusters, and reviewing policy results see the [Using Kubernetes Inventory](/docs/using/ui_usage/kubernetes/) section.
```
Comment (zhill) [inline content/docs/overview/concepts/images/inventory/_index.md:57]: Same here

**Review by zhill — APPROVED** [review]
To facilitate quick updates I'll merge this and then resolve the refs myself since I'm not sure if ref or relref is the right recommendation without some testing. But generally, links should use the hugo content xrefs: https://gohugo.io/content-management/cross-references/ so that it will manage updates properly and fail compile if not found.

**Pull request merged**